src/gen_modules_cvmix_idemix.F90

```diff
@@ -1,76 +1,76 @@
 !
 !
 !===============================================================================
-! module interface to FESOM2.0 for the CVMIX IDEMIX extension for the calculation 
+! module interface to FESOM2.0 for the CVMIX IDEMIX extension for the calculation
 ! of hte internal wave energy and its dissipationof in Turbulent Kinetic Energy
-! vertical mixing scheme --> Its based on the module interface for MPIOM programed 
-! by Nils Brüggeman & Oliver Gutjahr, This module calls subroutines from the 
-! CVMix library 
+! vertical mixing scheme --> Its based on the module interface for MPIOM programed
+! by Nils BrÃ¼ggeman & Oliver Gutjahr, This module calls subroutines from the
```

Comment:
The Family name is screwed up. I would replace to `Brueggeman`.

Reply:
I've restored this changes.